interfaces/webaudio.idl

```diff
@@ -90,6 +91,27 @@ dictionary AudioTimestamp {
   DOMHighResTimeStamp performanceTime;
 };
 
+[Exposed=Window]
+interface AudioRenderCapacity {
```

Comment:
I've filed https://github.com/WebAudio/web-audio-api/issues/2488 about issues here. It doesn't really block landing this PR though.